### PR TITLE
Add enactability property for singletons

### PR DESCRIPTION
**Pull request by WhatisRT**
# Description

While talking about an issue with conformance testing, I wasn't sure whether the enactability predicate was correct, so I proved this.

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [x] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [x] Self-reviewed the diff

### CODE REVIEW

File: src/Ledger/Gov.lagda [hunk 1]
```diff
@@ -3,14 +3,15 @@
 \begin{code}[hide]
 {-# OPTIONS --safe #-}
 
-open import Axiom.Set.Properties using (∃?-sublist-⇔)
 open import Ledger.Prelude hiding (any?; Any; all?; All; Rel; lookup; ∈-filter)
```
Comment (williamdemeo) [inline src/Ledger/Gov.lagda:6]: Why does this import appear before the module declaration?  Isn't this inconsistent with the rest of the library?

File: src/Ledger/Gov/Properties.agda [hunk 2]
```diff
@@ -1,21 +1,29 @@
 {-# OPTIONS --safe #-}
 
-import Data.List.Membership.Propositional as P
-open import Data.List.Membership.Propositional.Properties
-open import Data.List.Relation.Unary.Any hiding (map)
-
-open import Ledger.Prelude hiding (Any; any?)
```
Comment (williamdemeo) [inline src/Ledger/Gov/Properties.agda:7]: I'm in favor of moving these imports inside the module.

(although, this mixes a no-op/cosmetic mod into the PR... I'm fine with that, but have been told not to do it 😅 )

File: src/Ledger/Gov/Properties.agda [hunk 3]
```diff
@@ -146,3 +154,16 @@ instance
 
 Computational-GOV : Computational _⊢_⇀⦇_,GOV⦈_ (⊥ ⊎ String)
 Computational-GOV = it
+
+allEnactable-singleton : ∀ {aid s es} → getHash (s .prevAction) ≡ getHashES es (s .action)
+  → allEnactable es [ (aid , s) ]
+allEnactable-singleton {aid} {s} {es} eq = helper All.∷ All.[]
```
Comment (williamdemeo) [inline src/Ledger/Gov/Properties.agda:160]: Glad to see the enactability predicate is (likely) correct. 😅💦

**Review by williamdemeo — APPROVED** [review]
Nice one.

File: src/Ledger/Gov.lagda (hunk 1 above)
Comment (WhatisRT) [reply src/Ledger/Gov.lagda:6]: Ah, I missed that!

**Pull request merged**